modules/op-about-pipelinerun.adoc

```diff
@@ -0,0 +1,38 @@
+// This module is included in the following assembly:
+//
+// *openshift_pipelines/op-creating-applications-with-cicd-pipelines.adoc
+
+[id="about-pipelinerun_{context}"]
+= About PipelineRun
+
+A _PipelineRun_ instantiates a Pipeline for execution with specific inputs, outputs, and execution parameters on a cluster. A corresponding TaskRun is created for each Task automatically in the PipelineRun.
+
+All the Tasks in the Pipeline are executed in the defined sequence until all Tasks are successful or a Task fails. The `status` field tracks and stores the progress of each TaskRun in the PipelineRun for monitoring and auditing purpose.
+
+Here is an example of a PipelineRun to run a Pipeline `build-and-deploy` with relevant input and output parameters:
```

Comment:
`input and outpu parameters` doesn't sound correct... I would suggest simply `resources and parameters`